platform/android/MapboxGLAndroidSDK/proguard-rules.pro

```diff
@@ -1,11 +1,13 @@
 # By default, the flags in this file are appended to flags specified
 # in ../sdk/tools/proguard/proguard-android.txt,
 # contents of this file will be appended into proguard-android.txt
--keepattributes Signature, *Annotation*, EnclosingMethod
+-keepattributes Signature, *Annotation*, EnclosingMethod, InnerClasses, Exceptions
```

Comment:
Do we really need to keep attributes as InnerClasses and Exceptions? can this be simplified?

Reply:
I don't think we need, added them when testing things, good catch!